src/IssueViz.Security/Taint/TaintList/TaintIssuesControl.xaml

```diff
@@ -117,6 +117,14 @@
                         </DataTemplate>
                     </DataGridTemplateColumn.CellTemplate>
                 </DataGridTemplateColumn>
+                <DataGridTemplateColumn CanUserSort="True" SortMemberPath="TaintIssueViz.Issue.CreationTimestamp" Header="Created" Width="138"  MinWidth="100">
```

Comment:
Minor: "Created at" ? "Creation time"? Can be "created" as well, I can't think of a popular example to follow.